src/native/nb_native.gyp

```diff
@@ -7,7 +7,7 @@
             '-W',
             '-Wall',
             '-Wextra',
-            '-Werror',
+            # '-Werror',
```

Comment:
I would really prefer to keep failing on warnings for most of the code.
Wouldn't it be enough to disable warnings for third_party code?
For any code that we wrote we can easily fix warnings.
Check out this common_third_party.gypi and see maybe this can be done here:
https://github.com/noobaa/noobaa-core/blob/5d97fd2e7c97d7d778d69e11deb7730332dfa254/src/native/third_party/common_third_party.gypi#L4-L8